src/ui/buttons/props.js

```diff
@@ -914,6 +925,29 @@ export function normalizeButtonProps(
   }
 
   style = normalizeButtonStyle(props, style);
+  const { layout } = style;
+
+  const fundingSources = determineEligibleFunding({
+    fundingSource,
+    layout,
+    remembered,
+    platform,
+    fundingEligibility,
+    enableFunding,
+    components,
+    onShippingChange,
+    flow,
+    wallet,
+    applePaySupport,
+    supportsPopups,
+    supportedNativeBrowser,
+    experiment,
+    displayOnly,
+  });
```

Comment:
Can we leverage the same `renderedButtons` prop instead of recalculating this?

Reply:
just tried it - it appears to work, but flow notes 
> property `renderedButtons` is missing in  `ButtonPropsInputs` [1].Flow(prop-missing)

what would be the courteous way to handle this? Add `renderedButtons` to the type?

Reply:
Yeah that sounds appropriate